library/xml-model/src/test/resources/org/cru/godtools/xml/model/fallback_paragraph.xml

```diff
@@ -0,0 +1,7 @@
+<content:paragraph fallback="true" xmlns:content="https://mobile-content-api.cru.org/xmlns/content">
+    <content:image resource="image.png" restrictTo="web" />
+    <content:image resource="image.png" restrictTo="ios" />
+    <content:text>Test</content:text>
+    <content:text restrictTo="android">Test</content:text>
+    <content:text restrictTo="mobile">Test</content:text>
+</content:paragraph>
```

Comment:
So is this loading the paragraph from the api like all the other content which is just xml?

Reply:
this is a small snippet of content that is similar to the content the API returns. We are using it to test out the XML parser to ensure it correctly handles various parts of the content spec